build.jl

```diff
@@ -1,25 +1,31 @@
-using Images
+using Images, YAML, ColorTypes
+# Open the HTML prequel and put it into our content we build.
 f = open("open.txt")
+content = read(f,String)
```

Comment:
Not your fault, but this looks like it could use a `close(f)`. Or better, replace the whole `open/read/close` thing with just `content = read("open.txt", String)`. (Applies below too.)